lib/configs/angular.js

```diff
@@ -99,10 +98,16 @@ module.exports = {
         }],
         'space-before-blocks': [2, "always"],
         'consistent-this': [2, "self"],
-        'indent': [2, 4, {
-            SwitchCase: 1
-        }],
-        quotes: [2, "single"]
+        quotes: [2, "single"],
+        'prefer-const': 1,
```

Comment:
Also of note, this doesn't give us any perf performance. The only gain here is stylistic.